src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -146,73 +144,29 @@ public void shoot() {
 
   }
 
-  public void shootAuto() {
-    shooterHood.deployHood();
-
-    leader_shooterMAXRight.set(MoPrefs.getShooterFlywheelSetpoint());
-
-    if (shooterHood.hasReliableZero() && shooterHood.getFullyDeployed() && MoPrefs.getShooterFlywheelSetpoint()
-        - leader_shooterMAXRight.getEncoder().getVelocity() < MoPrefs.getShooterFlywheelTolerance()) {
-      shooterGate.set(MoPrefs.getShooterGateSetpoint());
-    } else {
-      shooterGate.set(0);
-    }
-  }
-
   public void idle() {
     // stop gate
     // slow shooter wheel
     shooterGate.stopMotor();
-
-    if (maintainFlywheelAtIdle) {
-      if (enablePID) {
-        shooterPIDRight.setReference(MoPrefs.getShooterFlywheelIdle(), ControlType.kVelocity);
-      } else {
-        leader_shooterMAXRight.set(MoPrefs.getShooterFlywheelIdle());
-      }
-    } else {
-      if (enablePID) {
-        shooterPIDRight.setReference(0, ControlType.kVelocity);
-      } else {
-        leader_shooterMAXRight.stopMotor();
-      }
-    }
+    leader_shooterMAXRight.stopMotor();
   }
 
   public void purge() {
     // stow hood
     // reverse gate
     // reverse shooter wheel
-    if (enablePID) {
-      shooterPIDRight.setReference(-1 * MoPrefs.getShooterFlywheelIdle(), ControlType.kVelocity);
-    } else {
-      leader_shooterMAXRight.set(-1 * MoPrefs.getShooterFlywheelIdle());
-    }
-
+    leader_shooterMAXRight.set(-0.2);
     shooterGate.set(-1 * MoPrefs.getShooterGateSetpoint());
   }
 
-  private void updateVelocity() {
-    double currPos = shooterEncoder.getPosition();
-    double currTime = Timer.getFPGATimestamp();
-    double dPos = currPos - lastPos;
-    double dT = currTime - lastTime;
-    lastPos = currPos;
-    lastTime = currTime;
-    currVelocity = dPos / dT * 60;
-  }
-
-  /**
-   * @return the currVelocity
-   */
-  public double getCurrVelocity() {
-    return currVelocity;
-  }
-
   @Override
   public void periodic() {
-    updateVelocity();
-    SmartDashboard.putNumber("Flywheel Speed", getCurrVelocity());
+    SmartDashboard.putNumber("Flywheel Speed", shooterEncoder.getVelocity());
     SmartDashboard.putNumber("Flywheel Position", shooterEncoder.getPosition());
+
+    NetworkTableEntry shooterPIDchooser = tab.add("Shooter PID", true).withWidget(BuiltInWidgets.kToggleSwitch)
```

Comment:
This can't be in periodic().  Adding the chooser can only happen once so it should be in init